pkg/config/trustengine.go

```diff
@@ -0,0 +1,51 @@
+package config
+
+import (
+	"errors"
+	"os"
+
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/trustengine"
+)
+
+// ResolveAllTrustEngineReferences retrieves all the step's secrets from the trust engine
+func ResolveAllTrustEngineReferences(config *StepConfig, params []StepParameters, trustEngineConfiguration trustengine.Configuration, client *piperhttp.Client) {
+	for _, param := range params {
+		if ref := param.GetReference("trustengineSecret"); ref != nil {
```

Comment:
Can we have `trustengineSecret` and in the future `trustengineSecretFile` as a constant somewhere?

Reply:
Done.